solidity/migrations/relay-config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "forceOnDemandSPV": false,
+  "forceTestnetRelay": false,
```

Comment:
I don't like having these two separate properties as we use them in the script in a manner similar to cases for a switch statement. Thoughts on having one property here with the following possibilities:
```json
  "forceRelay": "OnDemandSPV",
```
```json
  "forceRelay": "TestnetRelay",
```
By default, the property won't be provided in the `relay-config.json`, we will set this only when we need it.

Reply:
Good point! I will change.